pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.usort]
+
+first_party_detection = false
+
+[tool.black]
+
+line-length = 120
+target-version = ["py36"]
+
+[tool.ufmt]
```

Comment:
I would personally suggest to ignore the sphinx-gallery examples and to not format them, as we do in torchvision. But this is up to audio's maintainers to decide :)

Reply:
I. didn't find the `gallery` folder initially but I believe the equivalent here is  `examples/tutorials`. I decided to exclude the entire examples folder but happy to take other suggestions.